src/extension/LazyMint.sol

```diff
@@ -31,8 +31,8 @@ abstract contract LazyMint is ILazyMint, BatchMintMetadata {
    */
   function lazyMint(
     uint256 _amount,
-    string calldata _baseURIForTokens,
-    bytes calldata _data
+    string memory _baseURIForTokens,
```

Comment:
why the change from `calldata` to `memory`?

Reply:
This function is only invoked within the `CustomERC721::lazyMint` function, which, in turn, is only invoked by `CustomERC721::init`. The latter holds `_baseURIForTokens` and `_data` within its in-memory parameter `initPayload`.

Given this context, both `_baseURIForTokens` and `_data` already reside in memory and cannot be accessed as calldata-located data in solidity.

We can also change the declaration of the `CustomERC721::init` from `function init(bytes memory initPayload)` to `function init(bytes calldata initPayload)`.

Reply:
Okay cool thanks for the context @Yashiru !